libs/debugAssertApi/include/debugAssertApi/debugAssertApi.h

```diff
@@ -245,7 +245,7 @@ struct AutoRegisterAssertListener
 	AssertListener m_listener;
 };
 
-using AssertHandler = AssertResult(*)(const MsoAssertParams& params, const char* szMsg);
+using AssertHandler = int32_t(*)(const MsoAssertParams& params, const char* szMsg);
```

Comment:
>szMsg [](start = 76, length = 5)

(nit) perhaps this would be a good opportunity to rename this to non-hungarian :)